### PR TITLE
Run polling command fix

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://dfircommunity.slack.com/archives/CBEM7L3PV/p1612254917120100

## Description
Fix: Added string casting to string before striping.
Enhancement: Added encoding handling to the additional args

### CODE REVIEW

**Comment by ShahafBenYakir** [comment]
Failing UT that reproduces the issue. - https://app.circleci.com/pipelines/github/demisto/content/65133/workflows/261549b9-d2b6-452a-a9d1-3a9b862cb03b/jobs/268929
Run Unit Testing And Lint (268929) - demisto:content 2021-02-02 11-45-31

File: Packs/CommonScripts/ReleaseNotes/1_3_17.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### RunPollingCommand
+- Fixed script inputs parsing.
```
Comment (bakatzir) [inline Packs/CommonScripts/ReleaseNotes/1_3_17.md:4]:
```suggestion
- Fixed an issue where the script was not accepting integer inputs.
```

**Review by bakatzir — APPROVED** [review]

File: Packs/CommonScripts/ReleaseNotes/1_3_17.md (hunk 1 above)
Comment (ShahafBenYakir) [reply Packs/CommonScripts/ReleaseNotes/1_3_17.md:4]: It is not only this actually, also added encoding for the additional args inputs such that non-English chars could be accepted in the additional args.

File: Packs/CommonScripts/ReleaseNotes/1_3_17.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### RunPollingCommand
+- Fixed an issue where the script was failing on integers or non-english chars.
```
Comment (amshamah419) [inline Packs/CommonScripts/ReleaseNotes/1_3_17.md:4]:
```suggestion
- Fixed an issue where the script was failing when given integers or non-English characters as an argument.
```

**Comment by ShahafBenYakir** [comment]
Failed tests are not related to GP at all:
- ATD: error in submitting a file task
- Palo Alto Threat Vault: Test integration instance fails

Force merging

**Pull request merged**